skfp/fingerprints/topological_torsion.py

```diff
@@ -12,7 +12,97 @@
 
 
 class TopologicalTorsionFingerprint(BaseFingerprintTransformer):
-    """Topological torsion fingerprint."""
+    """
+    Topological Torsion fingerprint.
+
+    The implementation uses RDKit. This is a hashed fingerprint, where
+    the hashed fragments are computed based on topological torsions [1]_.
+
+    A topological torsion is defined as a sequence of 4 triplets:
+    (NPI-TYPE-NBR)-(NPI-TYPE-NBR)-(NPI-TYPE-NBR)-(NPI-TYPE-NBR)
+
+    It is a linear sequence of consecutively bonded non-hydrogen atoms where:
+    - NPI refers to the number of pi bond electrons for a given atom
+    - TYPE refers to the atomic type of given atom
+    - NBR is the number of non-hydrogen branches attached to it
```

Comment:
And if you rewrite this as graph invariant, similarly to Atom Pair (see comment below), you won't need those abbreviations